.github/workflows/pull-request-test.yml

```diff
@@ -126,46 +128,72 @@ jobs:
           test-timeout-mins: "120"
 
   test-pr-cypress:
-    runs-on: ubuntu-20.04
     if: ${{ github.event.action != 'closed' &&
       needs.check-deploy.outputs.pr-contains-string == 'true' &&
       (needs.check-deploy.outputs.test-enabled == 'true' ||
       needs.check-deploy.outputs.test-cypress-enabled == 'true') }}
     needs: [check-deploy, deploy-pr]
+    runs-on: ubuntu-22.04
+
+    strategy:
+      fail-fast: false
+      max-parallel: 1
+      matrix:
+        tests: [publicProject, updateProjects, useSession, rstudioSession]
+
     steps:
       - uses: actions/checkout@v3
       - uses: cypress-io/github-action@v5
-        name: Run Cypress acceptance tests
-        id: cypress
+        name: Verify infrastructure for Cypress acceptance tests
+        id: cypress-tests-preparation
         env:
+          BASE_URL: https://ci-renku-${{ github.event.number }}.dev.renku.ch
+          E2E_FOLDER: "cypress/e2e/"
+          E2E_SUFFIX: ".cy.ts"
           TEST_EMAIL: renku@datascience.ch
-          TEST_PASSWORD: ${{ secrets.RENKU_BOT_DEV_PASSWORD }}
           TEST_FIRST_NAME: Renku
           TEST_LAST_NAME: Bot
+          TEST_PASSWORD: ${{ secrets.RENKU_BOT_DEV_PASSWORD }}
           TEST_USERNAME: renku-test
+        with:
+          browser: chrome
+          install: true
+          spec: ${{ env.E2E_FOLDER }}verifyInfrastructure${{ env.E2E_SUFFIX }}
+          working-directory: cypress-tests
+      - uses: cypress-io/github-action@v5
+        name: Run Cypress acceptance tests in parallel
```

Comment:
Why is this test not using the same structure as above? (with `spec: ${{ env.E2E_FOLDER }}${{ matrix.tests }}${{ env.E2E_SUFFIX }}`)